app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -213,6 +220,11 @@ public void onReceive(Context context, Intent intent) {
 
                     // Run GC once to free all the garbage accumulated during provider initialization
                     System.gc();
+                } else if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.VANILLA_ICE_CREAM) {
+                    if (intent.getAction().equalsIgnoreCase(Intent.ACTION_PROFILE_AVAILABLE)
+                        || intent.getAction().equalsIgnoreCase(Intent.ACTION_PROFILE_UNAVAILABLE)) {
+                        privateSpaceStateEvent(intent.getParcelableExtra(Intent.EXTRA_USER, UserHandle.class));
+                    }
```

Comment:
Private space is just another profile like e.g. any work profile. So if I read this correctly, these change here is not needed.
KISS is already reacting on these intents in `ProfileChangedHandler`